specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json

```
@@ -43,7 +43,7 @@
         "operationId": "Table_Query",
         "x-ms-examples": {
           "TableGet": {
```

I assume you didn't change the property name as follows because it would be considered a breaking change, right?
```suggestion
          "TableQuery": {
```

---

I don't think examples can be breaking